Description: Include OpenVPN header file directory
 Compilation on Debian will fail if the header directory is not included.
Author: Troy Ready <troy@troyready.com>
Last-Update: 2017-08-24
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@
 all: plugin
 
 plugin: auth_script.c
-	$(CC) $(CFLAGS) $(LDFLAGS) -I. -c auth_script.c
+	$(CC) $(CFLAGS) $(LDFLAGS) -I. -I/usr/include/openvpn -c auth_script.c
 	$(CC) $(CFLAGS) $(LDFLAGS) -Wl,-soname,auth_script=.so -o auth_script.so auth_script.o
 
 clean:
